CE: Update tutorial to discourage PushMode.MANUAL usage (#127)
vaadin/docs: articles/ce/tutorial.asciidoc

}
----

Collaboration Engine works with any of the transport modes for push.
The default push mode, `PushMode.AUTOMATIC` should be used.

> Collaboration Engine does not behave correctly if you explicitely configure push to use `PushMode.MANUAL`, as Collaboraiton Engine does not invoke `UI.push()`, and the user interactions would therefore not be pushed out to the other connected users.

Reviewer: Fixed couple of typos.
Removed one instance of "Collaboration Engine".
Shortened the end a bit.
```suggestion
If you explicitly configure push to use `PushMode.MANUAL`, Collaboration Engine does not work, as it doesn't manually invoke `UI.push()` to send the user interactions to the other connected users.
```

> Collaboration Engine does not behave correctly if you explicitely configure push to use `PushMode.MANUAL`, as Collaboraiton Engine does not invoke `UI.push()`, and the user interactions would therefore not be pushed out to the other connected users.

Reviewer: Maybe it would be better to wrap this in a NOTE box, because it's not required to continue the tutorial, but just extra information?

Reviewer 2: If you do, remember to add a title for the admonition (note), which gives the gist, so that readers can quickly skip it if they are not interested in reading it all.

Author: @jouni can you check if I did it right? I took model from the "Subscription required for production use" IMPORTANT notice a bit above, but even by only copy-pasting the admonition to this location, GitHub preview wasn't able to display the header correctly anymore.

}
----

> .The default push mode, `PushMode.AUTOMATIC` should be used.

Reviewer 2: ```suggestion
.Manual push mode is not supported
```
IMO this would be easier to understand. Also no punctuation (periods, question/exclamation marks) in headings and titles.

Author: Doesn't this turn the message from more positive to negative?

Reviewer 2: Yes. I didn’t think that would be a problem. We would explicitly state what the limitation is, not try to glean over it. But if you feel strongly that you want to put a positive spin on it, I would go with `.Default push mode is recommended`
